src/components/Wheel/index.js

```diff
@@ -68,32 +55,37 @@ const Wheel = (props) => {
   const canvas = useRef(null);
 
   useEffect(() => {
-    const context = canvas.current.getContext('2d');
-    context.save();
-
-    context.scale(circleParameters.pixelRatio, circleParameters.pixelRatio);
-    context.fillStyle = 'rgba(255, 255, 255, 0.0)';
-    context.fillRect(0, 0, circleParameters.width, circleParameters.height);
-
-    context.translate(circleParameters.width / 2, circleParameters.height / 2);
-
-    context.lineWidth = '150';
-
-    fortune.forEach((data, index) => {
-      context.strokeStyle = data.color;
-      context.beginPath();
-      context.arc(
-        0,
-        0,
-        circleParameters.width / 3,
-        (Math.PI * 2 * index) / fortune.length,
-        (Math.PI * 2 * ++index) / fortune.length,
+    if (canvas.current !== null) {
+      const context = canvas.current.getContext('2d');
+      context.save();
+
+      context.scale(circleParameters.pixelRatio, circleParameters.pixelRatio);
+      context.fillStyle = 'rgba(255, 255, 255, 0.0)';
+      context.fillRect(0, 0, circleParameters.width, circleParameters.height);
+
+      context.translate(
+        circleParameters.width / 2,
+        circleParameters.height / 2,
       );
 
-      context.stroke();
-    });
+      context.lineWidth = '150';
+
+      fortune.forEach((data, index) => {
+        context.strokeStyle = data.color;
+        context.beginPath();
+        context.arc(
+          0,
+          0,
+          circleParameters.width / 3,
+          (Math.PI * 2 * index) / fortune.length,
+          (Math.PI * 2 * ++index) / fortune.length,
+        );
+
+        context.stroke();
+      });
 
-    context.restore();
+      context.restore();
+    }
   }, [
```

Comment:
Don't you need to add `canvas.current` here?

Reply:
Fixed. Updated in last commit